app/models/user.rb

```diff
@@ -289,6 +291,6 @@ def biography_limit
   end
 
   def send_devise_notification(notification, *args)
-    devise_mailer.send(notification, self, *args).deliver_later
+    devise_mailer.send(notification, self, *args).deliver_now
```

Comment:
Why do you need to change this?

Reply:
My mistake, I named this commit "For testing, remove this" but forgot to remove.